scripts/modules/commands.js

```diff
@@ -262,9 +262,6 @@ function registerCommands(programInstance) {
       const thresholdScore = parseFloat(options.threshold);
       const useResearch = options.research || false;
       
-      console.log(chalk.blue(`Analyzing task complexity from: ${tasksPath}`));
-      console.log(chalk.blue(`Output report will be saved to: ${outputPath}`));
-      
```

Comment:
I'm all for removing console logs if we're replacing them with other stuff, but right now it feels like we're just removing console logs for no reason.

Maybe we can add a progress push that would replace the console logs ?